visma/matrix/structure.py

```diff
@@ -29,6 +29,79 @@ def __init__(self):
         self.power = 1
         self.dim = [0, 0]
 
+    def __add__(self, other):
+        """Adds two matrices
+         Arguments:
+            self {visma.matrix.structure.Matrix} -- matrix token
+            other {visma.matrix.structure.Matrix} -- matrix token
+         Returns:
+            matSum {visma.matrix.structure.Matrix} -- sum matrix token
+         Note:
+            Make dimCheck before calling addMatrix
+        """
+        matSum = Matrix()
+        matSum.empty(self.dim)
+        for i in range(self.dim[0]):
+            for j in range(self.dim[1]):
+                matSum.value[i][j].extend(self.value[i][j])
```

Comment:
You can update the matrix classes since the base operations are overloaded.
```suggestion
                matSum.value[i][j] = self.value[i][j] + other.value[i][j]
```